React/tic-tac/src/app/components/Login/index.js

```diff
@@ -0,0 +1,48 @@
+import React, { Fragment, Component } from 'react';
+import { Redirect } from 'react-router-dom';
+
+import style from './style.scss';
+
+class Login extends Component {
+  state = {
+    loggedIn: false
+  };
+
+  loginHandle() {
+    this.setState(() => ({ loggedIn: true }));
+  }
+
+  fragmentRender() {
+    return (
+      <Fragment>
+        <div className={style.overlay}>
+          <div className={`${style.login} ${style.fadeInDown} ${style.animated}`}>
+            <input className={style.space} type="text" id="email" placeholder="E-mail" />
+            <input className={style.space} type="password" id="password" placeholder="Password" />
+            <input
+              className={style.space}
+              type="submit"
+              id={style.submit}
+              value="Log In"
+              onClick={() => this.loginHandle()}
+            />
+          </div>
+        </div>
+      </Fragment>
+    );
+  }
+
+  render() {
+    let redirectOrComponent;
+
+    if (this.state.loggedIn) {
+      redirectOrComponent = <Redirect to="/Game" />;
+    } else {
+      redirectOrComponent = this.fragmentRender();
+    }
+
+    return redirectOrComponent;
```

Comment:
You can validate the `loggedIn` bool here and return the component or a `Redirect` using a ternary expression.